app/test_main.py

```
@@ -1,3 +1,19 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+    ],
+)
+def test_get_human_age(cat_age: int, dog_age: int, expected: list[int]) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
```

This is a good set of test cases for the core functionality and the examples provided in the task description.

However, you're missing tests for some edge cases mentioned in the checklist (item #2 under "Clean Code"):

1.  **Negative numbers**: The function should be tested with negative inputs for `cat_age` and `dog_age`.
2.  **Incorrect data types**: The function should raise an exception (like `TypeError`) if it receives non-integer inputs (e.g., strings or floats). You can write a separate test for this using `pytest.raises`.

Adding these tests will make your test suite much more comprehensive.